test/v1/controllers/requestParsers/validators/validations/TaxYearValidationSpec.scala

```diff
@@ -75,5 +76,14 @@ class TaxYearValidationSpec extends UnitSpec with JsonErrorValidators {
 
     }
 
+    "the start year is below the minimum value" in {
+
+      val invalidTaxYear = "2019-20"
```

Comment:
Also you should probably use the taxYear just below the minimum which would be 2020-21